profiler/src/ProfilerEngine/Datadog.Profiler.Native/LibddprofExporter.cpp

```diff
@@ -45,32 +46,23 @@ std::string const LibddprofExporter::ProfilePeriodType = "RealTime";
 
 std::string const LibddprofExporter::ProfilePeriodUnit = "Nanoseconds";
 
-LibddprofExporter::LibddprofExporter(IConfiguration* configuration) :
-    _locationsAndLinesSize{512}
+LibddprofExporter::LibddprofExporter(IConfiguration* configuration, IApplicationStore* applicationStore) :
+    _locationsAndLinesSize{512}, _applicationStore{applicationStore}
```

Comment:
_locationsAndLinesSize{512}, 
_applicationStore{applicationStore}
```suggestion
    _locationsAndLinesSize{512}, _applicationStore{applicationStore}
    let's have one line for each field initialization for easier reading
```

Reply:
✔️